.github/workflows/docker-build-check.yml

```diff
@@ -0,0 +1,44 @@
+name: Docker Build Check
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
aside - @natikgadzhi - Wdyt of just running this for all commits/pushes? I can see this being a nice early warning for PRs in general.

Reply:
It's possible we want to build and even push out dev versions with commit sha as tag, but I'd talk about those when we feel the need. This one is so that my lazy ass doesn't merge something obviously broken by mistake ;)